es/account/memory.js

```diff
@@ -19,7 +19,11 @@
  * Memory Account module
  * @module @aeternity/aepp-sdk/es/account/memory
  * @export MemoryAccount
- * @example import MemoryAccount from '@aeternity/aepp-sdk/es/account/memory'
+ * @example //JS
+ * import MemoryAccount from '@aeternity/aepp-sdk/es/account/memory'
+ * 
+ * //NodeJS
+ * const { Universal: MemoryAccount } = require('@aeternity/aepp-sdk')
```

Comment:
```suggestion
 * const { MemoryAccount } = require('@aeternity/aepp-sdk')
```
MemoryAccount is not the same as Universal

Reply:
This looks inconsistent without other examples that show the same for each module. Shouldn't we update them by adding an example for nodejs? Or maybe instead we can remove all of them and write a general guide of amateurs developers how to import a part of sdk in different environments?

Reply:
This is just a first step for a bigger bounty campaign for contributing to the docs and is supposed to serve as one possible example. Inconsistencies are fine for now, as things will be unified in future work. Having a separate guide for environments is definitely a right thing to do, but for the sake of pragmatism and speed, it should be okay to have that extra line. People that come from google can get going right away without having to browse the documentation for additional setup information. For things that require the SDK to be set up we will have a short note about that.

Reply:
In my opinion, we don't need these examples, the guide folder already covers different import cases. Do you have ideas on how we can make the guides folder more accessible instead?